src/components/UtilityNav/UtilityNav.module.css

```diff
@@ -49,7 +49,8 @@
  * Utility nav link
  */
 .utility-nav__link {
-  @mixin textLink;
+  @mixin TextLinkInverted;
+  text-decoration: none;
```

Comment:
The link token was broken before so this was inheriting the text color; now the links are the same color as the nav background. I used the same styling solution as the primary nav: https://github.com/chanzuckerberg/edu-design-system/blob/next/src/components/PrimaryNav/PrimaryNav.module.css#L36